source/includes/_instruments.md

```diff
@@ -327,3 +327,165 @@ Http Status:
 | 400 Bad Request  | The request was malformed.  See response body    | None       | n/a              |
 | 401 Unauthorized | No auth provided, auth failed, or not authorized | None       | n/a              |
 | 404 Not Found    | No allocation with this UUID exists              | None       | n/a              |
+
+
+
+## Instrument Reporting
+
+The [`POST /instruments/reporting`](/#settlement-equity-post-instruments-reporting) endpoint allows platforms to report their system's view on equity ownership for reconcillation purposes.
```

Comment:
Be careful using the word ownership here. Not something I pretend to know lots about but there could be confusion in that for at least PP, this is _beneficial_ ownership.

Perhaps _allocation_ is simpler.

Reply:
sure, makes sense